modules/ROOT/pages/storage.adoc

```diff
@@ -0,0 +1,168 @@
+  filesystems:
+    - path: /var
+      device: /dev/disk/by-partlabel/var
+      # We can select the filesystem we'd like.
+      format: ext4
+      # Ask FCCT to generate a mount unit for us so that this filesystem gets
+      # mounted in the real root.
+      with_mount_unit: true
+----
+
+NOTE: The `start_mib` field is very important. In the future, we will make more clear how much space should be reserved for the root filesystem (see https://github.com/coreos/fedora-coreos-tracker/issues/586).
+
+You can of course mount only a subset of `/var` into a separate partition. For example, to mount `/var/lib/containers`:
+
+.Adding a /var/lib/containers partition to the primary disk
+[source,yaml]
+----
+variant: fcos
+version: 1.1.0
+storage:
+  disks:
+  - device: /dev/vda
+    wipe_table: false
+    partitions:
+    - size_mib: 0
+      start_mib: 5000
+      label: containers
+  filesystems:
+    - path: /var/lib/containers
+      device: /dev/disk/by-partlabel/containers
+      format: xfs
+      with_mount_unit: true
+----
+
+Alternatively, you can also mount storage from a separate disk. For example, here we mount `/var/log` from a partition on `/dev/vdb`:
+
+.Adding /var/log from a secondary disk
+[source,yaml]
+----
+variant: fcos
+version: 1.1.0
+storage:
+  disks:
+  - device: /dev/vdb
+    wipe_table: false
+    partitions:
+    - size_mib: 0
+      start_mib: 0
+      label: log
+  filesystems:
+    - path: /var/log
+      device: /dev/disk/by-partlabel/log
+      format: xfs
+      with_mount_unit: true
+----
+
+== Reconfiguring the root filesystem
+
+It is possible to reconfigure the root filesystem itself. You can use the path `/dev/disk/by-label/root` to refer to the original root partition. You must ensure that the new filesystem also has a label of `/root`.
+
+NOTE: You must have at least 4G of RAM for root reprovisioning to work.
+
+Here's an example of moving from xfs to btrfs, but reusing the same partition on the primary disk:
+
+.Changing the root filesystem to btrfs
+[source,yaml]
+----
+variant: fcos
+version: 1.1.0
+storage:
+  filesystems:
+    - device: /dev/disk/by-label/root
+      wipe_filesystem: true
+      format: btrfs
+      label: root
+----
+
+Similarly to the previous section, you can also move the root filesystem entirely. Here, we're moving root to a RAID1 device:
+
+.Moving the root filesystem to RAID1
+[source,yaml]
+----
+variant: fcos
+version: 1.1.0
+storage:
+  raid:
+    - name: root
+      level: raid1
+      devices:
+        - /dev/disk/by-id/virtio-disk1
+        - /dev/disk/by-id/virtio-disk2
+  filesystems:
+    - device: /dev/md/root
+      format: xfs
+      wipe_filesystem: true
+      label: root
+----
+
+NOTE: You don't need the `path` or `with_mount_unit` keys; FCOS knows that the root partition is special and will figure out how to find it and mount it.
+
+== Encrypted storage (LUKS)
+
+Here is an example to configure a LUKS device at `/var/lib/data`.
+
+[source.yaml]
```

Comment:
What would the secret be for such a device?

Reply:
The key-file is being generated by Ignition (The functionality was added in https://github.com/coreos/ignition/pull/1055)